src/main/webapp/user-page.html

```diff
@@ -138,6 +141,9 @@ <h6 class="font-weight-lighter text-muted" id="aboutme"></h6>
             Loading...
           </div>
         </div>
+          <div class="text-center add-marker-manually">
```

Comment:
This should probably be moved above the messages area - if there are too many messages, the user can miss this

Reply:
@anqitu let us know what you think since you're great at styling! 😄

Reply:
Sorry, but may I ask what is this button supposed to do?

Reply:
I thought the location is specific for each image?

Reply:
Sometimes when the location of the image can't be detected, it allows the users to add manually

Reply:
Maybe we can use the Map button on the post message div or/and add a button for each post message? I feel it is a bit confusing if we put the button there. As it should be for a single message, and not for a user profile config.

Reply:
Made the changes :)